Cross.toml

```diff
@@ -1,2 +1,5 @@
 [target.x86_64-unknown-linux-gnu]
-image = "ghcr.io/cross-rs/x86_64-unknown-linux-gnu:main-centos"
\ No newline at end of file
+image = "ghcr.io/cross-rs/x86_64-unknown-linux-gnu:main-centos"
+
+[target.x86_64-pc-windows-msvc]
+image = "ghcr.io/datadog/libdatadog/cross/x86_64-pc-windows-msvc-cross:testing-not-for-shippable-artifacts"
```

Comment:
why not for shippable artifacts ? what is the issue ?
should you consider `i686-pc-windows-msvc`. If the .NET profiler uses it at some point, this will be required (yeah we have customer with 32bits applications :) )

Reply:
Sorry for late response.
Its named "testing-not-for-shippable-artifacts" to just safeguard against folks actually using this to build artifacts for customers. The name was meant to be self descriptive. But it seems its confusing. I'll change it and add a note in Cross.toml